ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel/Wallets.hs

```diff
@@ -192,10 +192,10 @@ deleteHdWallet pw rootId = do
             Keystore.delete (WalletIdHdSeq rootId) (pw ^. walletKeystore)
             return $ Right ()
 
-removeKeyFromKeyStore :: PassiveWallet
+removeKeysFromKeyStore :: PassiveWallet
```

Comment:
`S` in `KeyStore` should be lowercase, because it's lowercase in all other occurrences of this word.